lib/transit_data.ex

```diff
@@ -0,0 +1,18 @@
+defmodule TransitData do
```

Comment:
Does this module serve a purpose?

Reply:
Ah, nope. It was generated by `mix new`. I'm never really sure what if anything is supposed to go in these root-level modules... I'll keep it but reduce its contents to just a moduledoc.